add_vector_to_matrix.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def add_vector_to_matrix(vec, matrix):
+    """
+    This function adds a given vector to each row of a given matrix.
+    :param vec: vector
+    :param matrix: matrix
+    :return: the matrix after added vector to each line
+    """
+    rows, cols = matrix.shape
+    for i in range(rows):
+        matrix[i, :] += vec
```

Comment:
What would happen if the vector given is in different length than the matrix's row?